homework/src/style.css

```diff
@@ -1,3 +1,124 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 16px;
+  background-color: #ffffff;
+  min-width: 300px;
+  margin-top: 0;
+}
+img{
+  border-radius: 3px;
+  height: 48px;
+}
+div.data {
+    display: inline-block;
+    padding-right: 25%;
+    padding-left: 25%;
+}
+
+.right-div{
+  display: inline-block;
+   
+}
+.header {
+  color: white;
+  background-color: rgb(63, 81, 181);
+  margin-bottom: 16px;
+  margin-top: 0;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px;
+  padding-top: 32px;
+  padding-right: 16px;
+  padding-bottom: 8px;
+  padding-left: 16px;
+  border-radius: 5px;
+}
+.table  {
+  border-spacing: 2px;
+  border-collapse: separate;
+  margin-bottom: 8px;
+  margin-left: 25%;
+  display: inline-block;
+  width: 29%;
+  background-color: rgb(255, 255, 255);
+  border-radius: 5px;
+  box-shadow: rgba(2, 1, 1, 0.2) 0px 1px 8px 0px,
+    rgba(0, 0, 0, 0.14) 0px 3px 4px 0px, rgba(0, 0, 0, 0.12) 4px 5px 5px -2px;
+  }
+
+#list {
+  margin-left: 16px;
+    font-size: 14px;
+    width: 250px;
+    height: 32px;
+    padding: 2px;
+}
 .alert-error {
-    color: red;
-}
\ No newline at end of file
+  color: #721c24;
+  background-color: #f8d7da;
+}
+
+.select {
+  font-size: 14px;
+  width: 100%;
+  
+}
+tr:first-child {
+    width: 100px;
+    min-width: 100px;
+    max-width: 100px;
+}
+tr {
+  font-weight: bold;
+}
+.contributor-list {
+  width: 20%;
+  display: inline-block;
+}
+.contributor {
+  font-size: 0.8rem;
+  color: #CEC0A6;
+}
+li {
+    border-bottom: solid 1px rgb(0, 0, 0, 12%);
+    padding: 16px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    cursor: pointer;
+}
+
+
+ul.contributor-list {
+    margin-bottom: 8px;
+    border: none;
+    border-radius: 2px;
+    margin-right: 45%;
+    margin-left: 25%;
+    background-color: #fff;
+    box-shadow: 0 1px 8px 0 rgba(0, 0, 0, .2), 0 3px 4px 0 rgba(0, 0, 0, .14), 0 3px 3px -2px rgba(0, 0, 0, .12);
+}
+div.contributors_badge{
+    font-size: 12px;
+    padding: 2px 8px;
+    line-height: 1rem;
+    background-color: gray;
+    color: white;
+    border-radius: 4px;
+}
+ @media only screen and (min-width:768px) and (max-width:1024px) {
+    .contributor .header .select {
+        width: 60%;
+        
+    }
+}
+@media (min-width: 1025px) {
+  .contributor .header .select {
+    width: 60%;
+  }
+}
```

Comment:
Your CSS sucks. The application does not look good and is not responsive:

On my computer screen:

![screenshot 2018-11-01 at 14 11 06](https://user-images.githubusercontent.com/2788771/47855648-81d0e380-dde5-11e8-8418-25f926ecb9af.png)

On an iPhone6/7/8:

![screenshot 2018-11-01 at 14 11 38](https://user-images.githubusercontent.com/2788771/47855673-8e553c00-dde5-11e8-8bef-0e60b736f2e6.png)

Reply:
It's done :)